ql/src/java/org/apache/hadoop/hive/ql/DriverContext.java

```diff
@@ -27,17 +29,25 @@
 import org.apache.hadoop.hive.metastore.api.TxnType;
 import org.apache.hadoop.hive.ql.cache.results.CacheUsage;
 import org.apache.hadoop.hive.ql.cache.results.QueryResultsCache.CacheEntry;
+import org.apache.hadoop.hive.ql.ddl.DDLSemanticAnalyzerFactory;
 import org.apache.hadoop.hive.ql.exec.FetchTask;
 import org.apache.hadoop.hive.ql.exec.Utilities;
+import org.apache.hadoop.hive.ql.exec.Task;
 import org.apache.hadoop.hive.ql.exec.tez.TezRuntimeContext;
 import org.apache.hadoop.hive.ql.exec.tez.TezTask;
 import org.apache.hadoop.hive.ql.lockmgr.HiveTxnManager;
+import org.apache.hadoop.hive.ql.parse.ASTNode;
+import org.apache.hadoop.hive.ql.parse.BaseSemanticAnalyzer;
+import org.apache.hadoop.hive.ql.parse.RewriteSemanticAnalyzer;
 import org.apache.hadoop.hive.ql.plan.mapper.StatsSource;
 
 /**
  * Context for the procedure managed by the Driver.
  */
 public class DriverContext {
+  public static final String DRIVER_DEFAULT_USER_NAME_PROP = "hive.driver.default.user.name";
```

Comment:
can we shorten the var name, it's already in DriverContext

Reply:
yeah, let me check